deployments/cloudfoundry/tile/tile.yml

```diff
@@ -0,0 +1,132 @@
+---
+# The high-level description of the tile.
+#
+name: splunk-opentelemetry-collector # By convention lowercase with dashes
```

Comment:
`splunk-otel-collector` is the convention

Reply:
The way the that tile interacts with the `splunk-otel-collector` job (defined by the BOSH release, `../bosh/jobs/splunk-otel-collector`) results in a naming conflict between the tile and the job.

For context, the `splunk-otel-collector` is the job that is actually deploying the collector internally.

Admittedly, the job itself is less visible to the user than the tile name itself, the job was just defined before the tile name so I let it keep its original name. However, if anyone's already running the BOSH release of the Splunk collector there may be some breaking change here if they've built any scripting around job name assumptions.

Do you think it's worth investigating impact and swapping names, or is there another name that's preferable?

Reply:
Do you have an example build failure?

Reply:
I just checked again and it ended up being successful. Not sure why it failed originally, but I'll update and use the proper naming convention since it's valid after all. Thanks!